Controller/PageAdminController.php

```diff
@@ -94,7 +94,9 @@ public function treeAction()
 
         $this->get('twig')->getExtension('form')->renderer->setTheme($formView, $this->admin->getFilterTheme());
 
-        return $this->render('SonataPageBundle:PageAdmin:tree.html.twig', array(
+        $templates = $this->retrieveAdminTemplates();
+
+        return $this->render($templates['tree'], array(
```

Comment:
Maybe something could be done here: https://github.com/sonata-project/SonataPageBundle/blob/42e550a0b2b561520ed233094b633a0d1b4706d2/Resources/config/admin.xml#L64-L68